vite.config.js

```diff
@@ -0,0 +1,35 @@
+// vite.config.js
+
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  build: {
+    lib: {
+      // the entry file that is loaded whenever someone imports
+      // your plugin in their app
+      entry: "src/index.ts",
+
+      // the exposed global variable
+      // is required when formats includes 'umd' or 'iife'
+      name: "HalJsonVuex",
+
+      // the proper extensions will be added, ie:
+      // name.js (es module)
+      // name.umd.cjs) (common js module)
+      // default fileName is the name option of package.json
+      fileName: "hal-json-vuex",
+    },
+    rollupOptions: {
+      // make sure to externalize deps that shouldn't be bundled
+      // into your library
+      external: ["vue"],
+      output: {
+        // Provide global variables to use in the UMD build
+        // for externalized deps
+        globals: {
+          vue: "Vue",
```

Comment:
Is this still necessary? Does Vue 3 still provide a global `Vue` object? You replaced many usages of `Vue.foo` in the code.

Reply:
Good point, I think it's not necessary anymore. Removed with https://github.com/ecamp/hal-json-vuex/pull/287/commits/f92ea43a9ee2f7e6d2b5a5c17d80dbad35cde97c